package.json

```diff
@@ -106,6 +106,8 @@
     "react-day-picker": "2.4.1",
     "react-docgen": "2.13.0",
     "react-dom": "15.4.0",
+    "react-facebook-login-component": "0.6.0",
```

Comment:
If we're going to replace these with our own buttons, I don't think we should import them into this project at all. What do you think about just using placeholder `<Button>` elements right now?